src/lib/PetitioRequest.ts

```diff
@@ -93,7 +92,7 @@ export class PetitioRequest {
 
 	/**
 	 * @param {*} dispatch The Undici agent or client instance you wish to use.
-	 * @param {*} keepAlive Whether to persist the dispatcher across requests or not.
+	 * @param {boolean} keepAlive Whether to persist the dispatcher across requests or not.
```

Comment:
Undo this please - we use `{*}` to avoid duplicating types since they're only necessary for JSDoc where types are not known by the compiler.